### ISSUE TITLE
Support for fetching specific country via URL

### ISSUE THREAD

**Issue by Mawreck** [pomber/covid19#46]
**Minor priority**

Would greatly increase performance. Then the fetching wont download via network all the countries.

Right now ->
https://pomber.github.io/covid19/timeseries.json

Could be ->
https://pomber.github.io/covid19/timeseries.json?country=thailand

**Comment by backtrackbaba** [comment]
Hi @Mawreck, I faced the same issue while I was working on a dashboard and had to slice and dice the data in a lot of ways. So I ended up consuming this time-series data and building a few queryable REST endpoints which consumes pomber's data and gives you only the data requested.

These endpoints not only give you country-wise data, but they also get you date-wise country and global data, time-series data in a date range and global cases as of today as well as any date.

Repo can be found [here](https://github.com/backtrackbaba/covid-api)
Docs can be found [here]( https://documenter.getpostman.com/view/2568274/SzS8rjbe?version=latest)

Let me know if this helps!
